File: Web Services/Cross/C# - Java/Servidor/AccesoDatos/AeropuertoDA.cs
using Modelo;
using MySql.Data.MySqlClient;
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace AccesoDatos
{
    public class AeropuertoDA
    {
        public int registrar(Aeropuerto a)
        {
            try
            {
                //Registrar el Driver
                
                string url = "server=quilla.lab.inf.pucp.edu.pe;" +
                    "user=a20151104;" +
                    "database=a20151104;" +
                    "port=3306;" +
                    "password=8Q8wTa;" +
                    "SslMode=none;";

                MySqlConnection con = new MySqlConnection(url);
                MySqlCommand stm = new MySqlCommand();
                con.Open();
                stm.Connection = con;
                stm.CommandText = "REGISTRAR_AEROPUERTO";
                stm.CommandType = System.Data.CommandType.StoredProcedure;

                stm.Parameters.Add("_CODIGO", MySqlDbType.VarChar).Value = a.Codigo;
                stm.Parameters.Add("_NOMBRE", MySqlDbType.VarChar).Value = a.Nombre;
                stm.Parameters.Add("_UBICACION", MySqlDbType.VarChar).Value = a.Ubicacion;
                stm.Parameters.Add("_MTS_ALTURA", MySqlDbType.Int32).Value = a.Mts_altura;
                stm.Parameters.Add("_ID_AEROPUERTO", MySqlDbType.Int32).Direction = System.Data.ParameterDirection.Output;

                stm.ExecuteNonQuery();
                a.Id = Int32.Parse(stm.Parameters["_ID_AEROPUERTO"].Value.ToString()) ;

                con.Close();
                return a.Id;
            }
            catch (Exception e)
            {
                return 0;
            }
        }
        public void modificar(Aeropuerto a)
        {
            try
            {
                string url = "server=quilla.lab.inf.pucp.edu.pe;" +
                   "user=a20151104;" +
                   "database=a20151104;" +
                   "port=3306;" +
                   "password=8Q8wTa;" +
                   "SslMode=none;";

                MySqlConnection con = new MySqlConnection(url);
                MySqlCommand stm = new MySqlCommand();
                con.Open();
                stm.Connection = con;
                stm.CommandText = "MODIFICAR_AEROPUERTO";
                stm.CommandType = System.Data.CommandType.StoredProcedure;

                //Registrar el Driver
                stm.Parameters.Add("_CODIGO", MySqlDbType.VarChar).Value = a.Codigo;
                stm.Parameters.Add("_NOMBRE", MySqlDbType.VarChar).Value = a.Nombre;
                stm.Parameters.Add("_UBICACION", MySqlDbType.VarChar).Value = a.Ubicacion;
                stm.Parameters.Add("_MTS_ALTURA", MySqlDbType.Int32).Value = a.Mts_altura;
                stm.Parameters.Add("_ID_AEROPUERTO", MySqlDbType.Int32).Value = a.Id;

                stm.ExecuteNonQuery();

                con.Close();
            }
            catch (Exception e)
            {

            }
        }

        public BindingList<Aeropuerto> listar()
        {
            BindingList<Aeropuerto> lista = new BindingList<Aeropuerto>();
            try
            {
                string url = "server=quilla.lab.inf.pucp.edu.pe;" +
                   "user=a20151104;" +
                   "database=a20151104;" +
                   "port=3306;" +
                   "password=8Q8wTa;" +
                   "SslMode=none;";

                MySqlConnection con = new MySqlConnection(url);
                MySqlCommand stm = new MySqlCommand();
                con.Open();
                stm.Connection = con;
                stm.CommandText = "LISTAR_AEROPUERTOS_COMPLETO";
                stm.CommandType = System.Data.CommandType.StoredProcedure;

                MySqlDataReader rs = stm.ExecuteReader();

                while (rs.Read())
                {
                    //agregar elementos a lista
                    Aeropuerto p = new Aeropuerto();

                    p.Id = rs.GetInt32("ID_AEROPUERTO");
                    p.Codigo = rs.GetString("CODIGO");
                    p.Nombre = rs.GetString("NOMBRE");
                    p.Ubicacion = rs.GetString("UBICACION");
                    p.Mts_altura = rs.GetInt32("MTS_ALTURA");

                    lista.Add(p);
                }
                con.Close();
                return lista;
            }
            catch (Exception ex)
            {
                return null;
            }
        }

        public void eliminar(int id)
        {
            try
            {
                string url = "server=quilla.lab.inf.pucp.edu.pe;" +
                   "user=a20151104;" +
                   "database=a20151104;" +
                   "port=3306;" +
                   "password=8Q8wTa;" +
                   "SslMode=none;";

                MySqlConnection con = new MySqlConnection(url);
                MySqlCommand stm = new MySqlCommand();
                con.Open();
                stm.Connection = con;
                stm.CommandText = "ELIMINAR_AEROPUERTO";
                stm.CommandType = System.Data.CommandType.StoredProcedure;

                //Registrar el Driver

                stm.Parameters.Add("_ID_AEROPUERTO", MySqlDbType.Int32).Value = id;

                stm.ExecuteNonQuery();

                con.Close();
            }
            catch (Exception e)
            {

            }
        }
    }
}

File: Web Services/Cross/Java - C#/Servidor/src/java/AccesoDatos/AeropuertoDA.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package AccesoDatos;

import Modelo.Aeropuerto;
import java.sql.CallableStatement;
import java.sql.Connection;
import java.sql.DriverManager;
import java.sql.ResultSet;
import java.util.ArrayList;

/**
 *
 * @author Alvaro
 */
public class AeropuertoDA {
    public int registrar(Aeropuerto a){
        try{
            //Registrar el Driver
            Class.forName("com.mysql.jdbc.Driver");
            String url="jdbc:mysql://quilla.lab.inf.pucp.edu.pe/a20151104";
            
            Connection con = DriverManager.getConnection(url,"a20151104","8Q8wTa");
            CallableStatement stm = con.prepareCall("{call REGISTRAR_AEROPUERTO(?,?,?,?,?)}");
            
            stm.setString("_CODIGO",a.getCodigo());
            stm.setString("_NOMBRE",a.getNombre());
            stm.setString("_UBICACION",a.getUbicacion());
            stm.setInt("_MTS_ALTURA",a.getMts_altura());
            stm.registerOutParameter("_ID_AEROPUERTO", java.sql.Types.INTEGER);
            stm.execute();
            a.setId(stm.getInt("_ID_AEROPUERTO"));
            //System.out.println("Registro realizado");
            con.close();
            return a.getId();
        }catch(Exception e){
            //System.out.println(e.getMessage());
            return 0;
        }
    }
    public void modificar(Aeropuerto a){
        try{
            //Registrar el Driver
            Class.forName("com.mysql.jdbc.Driver");
            String url="jdbc:mysql://quilla.lab.inf.pucp.edu.pe/a20151104";
            
            Connection con = DriverManager.getConnection(url,"a20151104","8Q8wTa");
            CallableStatement stm = con.prepareCall("{call MODIFICAR_AEROPUERTO(?,?,?,?,?)}");
            
            stm.setInt("_ID_AEROPUERTO",a.getId());
            stm.setString("_CODIGO",a.getCodigo());
            stm.setString("_NOMBRE",a.getNombre());
            stm.setString("_UBICACION",a.getUbicacion());
            stm.setInt("_MTS_ALTURA",a.getMts_altura());
            stm.execute();
          
            //System.out.println("Registro realizado");
            con.close();
        }catch(Exception e){
            //System.out.println(e.getMessage());
        }
    }    
    public ArrayList<Aeropuerto> listar(){
        ArrayList<Aeropuerto> lista = new ArrayList<Aeropuerto>();
        try{
            Class.forName("com.mysql.jdbc.Driver");
            String url="jdbc:mysql://quilla.lab.inf.pucp.edu.pe/a20151104";
            Connection con = DriverManager.getConnection(url,"a20151104","8Q8wTa");
            CallableStatement stm = con.prepareCall("{call LISTAR_AEROPUERTOS_COMPLETO()}");

            ResultSet rs = stm.executeQuery();
            while(rs.next()){
                //agregar elementos a lista
                Aeropuerto p = new Aeropuerto();
                p.setId(rs.getInt("ID_AEROPUERTO"));
                p.setCodigo(rs.getString("CODIGO"));
                p.setNombre(rs.getString("NOMBRE"));
                p.setUbicacion(rs.getString("UBICACION"));
                p.setMts_altura(rs.getInt("MTS_ALTURA"));
                lista.add(p);
            }
            con.close();
        }catch(Exception ex){
            
        }
        return lista;
    }
    
    public void eliminar(int id){
        try{
            //Registrar el Driver
            Class.forName("com.mysql.jdbc.Driver");
            String url="jdbc:mysql://quilla.lab.inf.pucp.edu.pe/a20151104";
            
            Connection con = DriverManager.getConnection(url,"a20151104","8Q8wTa");
            CallableStatement stm = con.prepareCall("{call ELIMINAR_AEROPUERTO(?)}");
            
            stm.setInt("_ID_AEROPUERTO",id);
            stm.execute();
          
            //System.out.println("Registro realizado");
            con.close();
        }catch(Exception e){
            //System.out.println(e.getMessage());
        }
    }
}